pkg/rpc/server/server_test.go

```diff
@@ -48,18 +48,25 @@ type rpcTestCase struct {
 	check  func(t *testing.T, e *executor, result interface{})
 }
 
-const testContractHash = "33f3421677fab7f620bd70582f468b4a18df1e5d"
+const (
+	testContractHash        = "33f3421677fab7f620bd70582f468b4a18df1e5d"
+	testInvocationTxHash    = "2a98062dc156be04b6be37aed065041287d16845c5500c6d8ff4c328f8bfe191"
+	testNativeGASHash       = "3b7d3711c6f0ccf9b1dca903d1bfa1d896f1238c"
+	testSendRublesTxHash    = "14851349ba853f16c29cb7ff4b43aab63a2050842efecda36c4667ca676aa2d2"
+	testRecieveRublesTxHash = "4452090ccb876ef87ffbc5011208e4eeab008d3693a13be480689e51096be0f2"
+	testRecieveGASTxHash    = "eeb5743b15f60a2efa48b91153ac70bf7718a8c28dd5865bb40c53895b1f702b"
```

Comment:
Somehow we managed to live without many of these constants, what has really changed?

Reply:
Some of them are being used multiple times in our test code, so I decided that it would be more convenient  to move them into separate place. Move them back?

Reply:
Probably not, I'll check this once.